include/ofi_util.h

```diff
@@ -801,10 +801,10 @@ struct util_eq {
 
 struct util_event {
 	struct slist_entry	entry;
-	int			size;
+	ssize_t			size;
```

Comment:
This change lets us fix up a couple of unneeded casts now.  See util_eq.c, lines 107 and 146

Reply:
Ah, I didn't check and also didn't expect casts there. I removed the casts except of util_eq.c:112 (ret = MIN(len, (size_t)entry->size)), because I guess compilers would complain about signed vs unsigned comparison.